packages/components/src/components/post-avatar/post-avatar.tsx

```diff
@@ -60,6 +60,21 @@ export class PostAvatar {
     checkNonEmpty(this, 'firstname');
   }
 
+  @Watch('lastname')
+  validateLastname() {
+    checkNonEmpty(this, 'lastname', undefined, 'warning');
+  }
+
+  @Watch('userid')
+  validateUserid() {
+    checkNonEmpty(this, 'userid', undefined, 'warning');
+  }
+
+  @Watch('email')
+  validateEmail() {
+    checkNonEmpty(this, 'email', undefined, 'warning');
+  }
```

Comment:
These properties are declared with the `?:` notation which means they are optional and therefore can be empty. To validate them, you can use the `checkEmptyOrType()` validator instead.